controllers/array_action/array_mediator_ds8k.py

```diff
@@ -685,18 +685,33 @@ def get_flashcopy_state(self, flashcopy_id):
     def get_replication(self, volume_internal_id, other_volume_internal_id, other_system_id):
         raise NotImplementedError
 
+    def get_ear_replication(self, volume_internal_id):
```

Comment:
I dont believe ds8k will ever have EAR

Reply:
But it is raising not implemented error

Reply:
suggesting it will be implemented in the future..